apis/taskcluster.py

```diff
@@ -18,6 +20,105 @@
 TRIGGER_TOTAL = 4
 
 
+# These are intentionally ordered so that the ones that override the others have a higher value
+@unique
+class Classification(IntEnum):
+    NotYourFault = 4
+    NewFailure = 3
+    PossibleIntermittent = 2
+    Unassigned = 1
+
+    def mini(self):
+        if self.value == Classification.NotYourFault:
+            return "NYF"
+        elif self.value == Classification.NewFailure:
+            return "F"
+        elif self.value == Classification.PossibleIntermittent:
+            return "I"
+        elif self.value == Classification.Unassigned:
+            return "U"
+
+    @staticmethod
+    def from_string(s, logger):
+        if s == "New Failure":  # from Push Health
+            return Classification.NewFailure
+        elif s == "new failure not classified":  # from TC
+            return Classification.NewFailure
+        elif s == "fixedByCommit":  # from Push Health
+            return Classification.NotYourFault
+        elif s in ["autoclassified intermittent", "expected fail", "fixed by commit", "infra"]:  # from TC
+            return Classification.NotYourFault
+        elif s == "intermittent":  # from Push Health
+            return Classification.PossibleIntermittent
+        elif s in ["intermittent", "not classified"]:  # from TC
+            return Classification.PossibleIntermittent
```

Comment:
The `intermittent` check is kind of redundant here. But I guess to convey that it comes from either Push Health or TC that is okay.